Tickets/DM-36041: Updates to alert-stream-broker user manual

bsmartradio opened the pull request:
Update the links. Additionally add new troubleshooting comments, the recommended startup sequence in Argo, and suggestions on the configuration of Kafka and how//where to edit specific values.

bsmartradio committed:
Update Phalanx links in guide

bsmartradio committed:
Add start-up order and adjusted DNS instructions

DNS instructions for broker IPs were out of date, as we no longer use kubectl to privision IPs. This is now done by Square. Added a startup order that is helpful in troubleshoot problematic parts of alert-broker.

ebellm commented on index.rst:
```diff
@@ -49,23 +49,61 @@ When "prune" is enabled, Argo will delete any orphaned resources that no longer
+If for some reason the instance of alert-stream-broker has been removed from the active applications, you can re-deploy it by going to the
+`science-platform`_ application and re-syncing alert-stream-broker.
+
+Trouble shooting: If all of the brokers have faield but everything else is running, the brokers may be out of storage.
```
```suggestion
Trouble shooting: If all of the brokers have failed but everything else is running, the brokers may be out of storage.
```

ebellm reviewed (approved):
one tiny typo

bsmartradio committed:
Add troubleshooting info and update all links

Added a number of troubleshooting sections when trying to setup the alert-stream-broker andupdated several broken links.
Typo fixes
